packages/venia-ui/lib/queries/userSubscription.graphql

```diff
@@ -0,0 +1,9 @@
+# expects bearer header to be set via context to return data
+mutation userSubscription($is_subscribed: Boolean!) {
```

Comment:
Buried in our wiki we have documented our preference for GraphQL colocation and naming. This variable name can also be camelCased so its easier to reference in code.

https://github.com/magento/pwa-studio/wiki/Project-coding-standards-and-conventions#graphql-file-location-and-naming

```suggestion
mutation SET_NEWSLETTER_SUBSCRITION($isSubscribed: Boolean!) {
```